software/bracket.yml

```
@@ -0,0 +1,13 @@
+name: "Bracket"
+website_url: "https://evroon.github.io/bracket"
+source_code_url: "https://github.com/evroon/bracket"
+description: "Flexible tournament system to build a tournament setup, add teams, schedule matches, keep track of scores and present ranking live to the public."
+licenses:
+  - MIT
+platforms:
+  - Python
+  - Javascript
+  - Docker
+tags:
+  - Miscellaneous
+depends_3rdparty: false
```

Optional field so can be removed 
```suggestion
  - Miscellaneous
```